fast64_internal/sm64/settings/panels.py

```diff
@@ -0,0 +1,55 @@
+import os
+from bpy.utils import register_class, unregister_class
+from bpy.types import Scene, Context, UILayout
+
+from ...utility import prop_split
+from ...panels import SM64_Panel
+
+
+def draw_repo_settings(scene: Scene, layout: UILayout):
+    col = layout.column()
+    sm64_props = scene.fast64.sm64
+
+    col.prop(
+        sm64_props,
+        "repo_settings_tab",
+        text="Repo Settings",
+        icon="TRIA_DOWN" if sm64_props.repo_settings_tab else "TRIA_RIGHT",
+    )
+    if not sm64_props.repo_settings_tab:
+        return
+
+    prop_split(col, sm64_props, "compression_format", "Compression Format")
+    prop_split(col, sm64_props, "refresh_version", "Refresh (Function Map)")
+
+
+class SM64_GeneralSettingsPanel(SM64_Panel):
+    bl_idname = "SM64_PT_general_settings"
+    bl_label = "SM64 General Settings"
+
+    def draw(self, context: Context):
+        col = self.layout.column()
+        sm64_props = context.scene.fast64.sm64
+        sm64_props = context.scene.fast64.sm64
+
+        if sm64_props.export_type == "C":
+            # If the repo settings tab is open, we set show_repo_settings to false
```

Comment:
```suggestion
            # If the repo settings tab is open, we pass show_repo_settings as False
```
I was confused and looking for a `show_repo_settings` property anywhere